compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIESplitLogicalObjFifos.cpp

```diff
@@ -138,6 +177,9 @@ FailureOr<int64_t> getSplitStride(ArrayRef<AMDAIE::DmaCpyNdOp> dmaOps,
 /// that has product size larger than the other side's product size after
 /// splitting because that's the number of elements that should be
 /// produced/consumed on the respective sides before splitting.
+/// Towards the end fetch the count of unique L2<->L1 for the objectFifo which
```

Comment:
Remove mentions related to L1/L2 from the comments, naming etc. This is not about L1/L2 connections but about producer/consumer connections.